config/channel-settings.json

```diff
@@ -108,6 +116,28 @@
         "configModel": {
           "type": "object",
           "properties": {
+            "fileType": {
+              "title": "File Type",
+              "type": "string",
+              "enum": [
+                "Delimited",
+                "XML"
+              ]
+            },
+            "deferred": {
```

Comment:
I think tony had `putDeferred` in the documentation... I think b/c there was uncertainty if we may need `getDeferred` in the future.